README.rst

```diff
@@ -77,21 +78,99 @@ Update
 
 On every new version, reinstall the dependencies since there might be new ones.
 
-Themes
-------
+Themes and styling
+------------------
 
 To try out class-less themes use the context processor
 ``argus_htmx.context_processor.theme_via_session`` instead of
 ``argus_htmx.context_processor.theme_via_GET``.
 
-There are plenty of themes copied from `CSS Bed`_ to play with. Your own themes
-can be added by copying one css-file per theme to the static path ``themes/``,
-either in an app or in a directory mentioned in ``STATICFILES_DIRS``.
+Default included themes are: `light`, `dark` and `argus`.
+
+This project supports Tailwind CSS utility classes and daisyUI components for styling.
+Below is an overview of the stack, installation and build instructions, and configuration details for themes and styles.
+
+Overview
+~~~~~~~~
+* Tailwind CSS: A utility-first CSS framework for rapidly building custom user interfaces.
+* daisyUI: A component library for Tailwind CSS that provides a set of ready-to-use components as well as color themes.
+
+Installation and build instructions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Recommended but open for tweaks and adaptations steps:
+
+1. Install Tailwind standalone CLI bundled with daisyUI from https://github.com/dobicinaitis/tailwind-cli-extra as ``tailwindcss`` executable. Optionally you can compile tailwind+daisyUI standalone cli bundle yourself as described here: https://github.com/tailwindlabs/tailwindcss/discussions/12294#discussioncomment-8268378.
+2. Generate Tailwind config file by running ``tailwind_config`` management command. By default the generated file will be based on ``src/argus_htmx/tailwindtheme/tailwind.config.template.js`` and expected values will be injected with reasonable defaults. For customization options see `Customization`_.
+3. Build main stylesheet file using ``tailwindcss`` executable from step 1 and pointing to config file from step 2::
+
+    ./tailwindcss -c tailwind.config.js -i ./styles.css --output ../static/styles.css
+   optional ``--watch`` flag may be included if you want the ``--output`` CSS file being rebuild on changes in template files.
+
+
+Customization
+~~~~~~~~~~~~~
+Following are a subject to customization:
+
```

Comment:
All of this section should go in a separate file. It is not daily bread and butter dev. In another PR.